tests/TestControls/tst_test-statusinput.qml

```diff
@@ -32,6 +34,13 @@ Item {
 
         when: windowShown
 
+        //> Simulate key and wait for side effects
+        function pressKeyAndWait(key, time) {
```

Comment:
will this be usefull for many test? I think we just duplicated this function, maybe we can extract this?

Reply:
Done. Thanks